src/elements/content-sidebar/activity-feed/comment/__tests__/BaseComment.test.js

```diff
@@ -0,0 +1,273 @@
+// @flow
+
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { fireEvent, render, screen } from '@testing-library/react';
+import BaseComment from '../BaseComment';
+
+jest.mock('../../Avatar', () => () => 'Avatar');
+jest.mock('react-intl', () => ({
+    ...jest.requireActual('react-intl'),
+    FormattedMessage: ({ defaultMessage }: { defaultMessage: string }) => <span>{defaultMessage}</span>,
+}));
+
+const getUserProfileUrl = jest.fn().mockResolvedValue('https://www.test.com/');
+
+const currentUser = {
+    name: 'testuser',
+    id: 11,
+};
+
+const approverSelectorContacts = [];
+const mentionSelectorContacts = [];
```

Comment:
I see this was in the original test but I don't think it provides much value to have these are separate variables instead of having them inline in the component in `getWrapper`. this is partly because the variable names aren't any more descriptive than the prop name itself and I think we're just passing these to satisfy the prop type requirements

they're only used once so it would be easier to understand the following code:
```
const getWrapper = props =>
    render(
        <IntlProvider locale="en">
            <BaseComment
                [...]
                approverSelectorContacts={[]}
                mentionSelectorContacts={[]}
                [...]
            />
        </IntlProvider>,
    );
```
than having to locate the variables